data/org.mate.pluma.gschema.xml.in

```diff
@@ -216,5 +216,25 @@
       <summary>Active plugins</summary>
       <description>List of active plugins. It contains the "Location" of the active plugins. See the .pluma-plugin file for obtaining the "Location" of a given plugin.</description>
     </key>
+    <key name="enable-space-drawer-space" type="i">
+      <default>0</default>
+      <summary>Draw spaces</summary>
+      <description>Whether pluma should draw spaces in the editor window: 0 no drawing; 1 drawing only trailing spaces; 2 drawing all spaces.</description>
+    </key>
+    <key name="enable-space-drawer-tab" type="i">
+      <default>0</default>
+      <summary>Draw tabs</summary>
+      <description>Whether pluma should draw tabs in the editor window: 0 no drawing; 1 drawing only trailing tabs; 2 drawing all tabs.</description>
+    </key>
+    <key name="enable-space-drawer-newline" type="b">
+      <default>false</default>
+      <summary>Draw newline</summary>
+      <description>Whether pluma should draw newlines in the editor window.</description>
+    </key>
+    <key name="enable-space-drawer-nbsp" type="i">
```

Comment:
All these options should use `type="enum"` rather than `int`. Take a look at `org.mate.marco.gschema.xml` to see how they do it, but it amounts to something like:
```xml
<schemalist>
...
  <enum id="org.mate.pluma.SpaceDrawer">
    <value nick="draw-none" value="0"/>
    <value nick="draw-trailing" value="1"/>
    <value nick="draw-all" value="2"/>
  </enum>
...
  <schema id="...">
...
  <key name="enable-space-drawer-nbsp" type="org.mate.pluma.SpaceDrawer">
    <default>'draw-none'</default>
    ...
  </key>
...
```
(and update the `<description>` accordingly)

Reply:
I did as your request, which seems more understatable